RoddyCore/src/de/dkfz/roddy/execution/io/SSHExecutionService.groovy

```diff
@@ -131,18 +158,43 @@ class SSHExecutionService extends RemoteExecutionService {
 
                 c.startSession()
                 t2 = System.nanoTime()
-                logger.postSometimesInfo(RoddyIOHelperMethods.printTimingInfo("start ssh client session", t1, t2))
+                logger.sometimes(RoddyIOHelperMethods.printTimingInfo("start ssh client session", t1, t2))
+            } catch (UserAuthException ex) {
+                logger.severe(
+                        [
+                                "Could not setup SSH access with your configuration:",
+                                "- Is the user and password combination right?",
+                                "- Did you store the password and is it right?",
+                                "- Did you use keyfile authentification and are there any keyfiles available and accessible?",
+                                "- Did you set the custom keyfile location and is this file accessible?",
+                                "- Did you try to use an ssh agent and is it setup properly? Was Roddy called properly?",
+                                "- Did you check if the authentification method is right (keyfile, password, sshagent)?"
+                        ].join("\n\t")
+                )
+
+                Roddy.exit(1)
             } catch (Exception ex) {
```

Comment:
Is this really the place to catch all exceptions? Should that not better be done further up the stack? The Roddy.exit (singleton!) creates a strong dependency on the Roddy context.